app/stories/shared.stories.jsx

```diff
@@ -341,3 +345,28 @@ story('Sortable list')
     .addWithJSX('field list', () => (
         <FieldListContainer />
     ))
+
+story('BackButton')
+    .addDecorator(StoryRouter())
+    .addWithJSX('basic', () => (
+        <div>
+            <BackButton />
+            <hr />
+        </div>
+    ))
+
+story('SvgIcon')
+    .addWithJSX('all', () => (
+        <Row>
+            {SvgIcon.names.map((name) => (
```

Comment:
@juhah @tumppi here's how I did it. All icons that are defined via `SvgIcon` are gonna be automatically rendered here. It'd be great if we use `viewBox` on all SVGs.. that'd make them scalable.